features/data/project/flask_py36/Pipfile

```
@@ -9,4 +9,4 @@ verify_ssl = true
 flask = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.9"
```

why `py3.9`?

---

seems like a mistake.
```suggestion
python_version = "3.6"
```